apps/src/signUpFlow/LoginTypeSelection.tsx

```diff
@@ -1,13 +1,14 @@
 import React, {useState, useEffect} from 'react';
 
-import {Button} from '@cdo/apps/componentLibrary/button';
+import {Button as NewButton} from '@cdo/apps/componentLibrary/button';
 import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon';
 import TextField from '@cdo/apps/componentLibrary/textField/TextField';
 import {Heading3, BodyThreeText} from '@cdo/apps/componentLibrary/typography';
+import Button from '@cdo/apps/legacySharedComponents/Button';
```

Comment:
This is definitely a nit and not blocking but wondering if it would be better to call this `LegacyButton` and keep the component library button as `Button` to make it clearer which one should be preferred

Reply:
Either way, can you add a comment about why it's needed?